src/styles/base/_base.scss

```diff
@@ -0,0 +1,21 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 10px;
+
+  @extend %font-roboto;
```

Comment:
It seems like you're trying to use a Sass placeholder selector (`%font-roboto`) but there's no indication that this placeholder was defined in the `_placeholders.scss` file. Make sure that `%font-roboto` is defined properly in the placeholders file and is included in the project to avoid any compilation errors.